moodle/Util/Tokens.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+// This file is part of Moodle - https://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <https://www.gnu.org/licenses/>.
+
+namespace MoodleHQ\MoodleCS\moodle\Util;
+
+use PHP_CodeSniffer\Files\File;
+use PHPCSUtils\Utils\ObjectDeclarations;
+
+class Tokens
```

Comment:
Uhm, I think this is the only detail that I don't like too much. PHP_CodeSniffer already has `PHP_CodeSniffer\Util\Tokens` that is imported by various Sniffs as `Tokens`.
And that may cause confusion sometimes... maybe `TokenUtil` (coz it's similar to existing `MoodleUtil`) may be good enough to avoid the dupes?

Reply:
Good call. Done.